lib/rules/require-class-comment.js

```diff
@@ -331,6 +331,13 @@ module.exports = function(context) {
 				});
 			}
 
+			if (!isVirtual && isAbstract) {
```

Comment:
It looks like those booleans have really bad names :smile:

In an ideal world, it should be `isVirtual && !isAbstract`